client/etcd/config.go

```diff
@@ -30,6 +30,9 @@ import (
 type ClusterConfig struct {
 	Zone      string   `yaml:"zone"`
 	Endpoints []string `yaml:"endpoints"`
+	Cert      string   `yaml:"cert",omitempty`
```

Comment:
nit: group these fields into a struct and add a suffix to indicate these string refer to a path. E.g. https://github.com/m3db/m3em/blob/master/services/m3em_agent/config/config.go#L67-L72

Reply:
nit: probably better to do `TLS *TLSConfiguration` to indicate it's optional